### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by peter-plochan**
* Add modules section
* Add javadoc-model badge

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: README.md [hunk 1]
```diff
@@ -10,6 +11,12 @@ The *GoodData Java SDK* encapsulates the REST API of the **GoodData Platform**.
 The first version was implemented during the [All Data Hackathon](http://hackathon.gooddata.com) April 10 - 11 2014.
 It is free and open-source software provided "as-is" under the [BSD License](LICENSE.txt) as an official project by [GoodData Corporation](http://www.gooddata.com).
 
+## Modules
+
+The *GoodData Java SDK* contains following modules:
+* [gooddata-java-model](./gooddata-java-model) - Lightweight library containing only GoodData API structures.
+* [gooddata-java](./gooddata-java) - The main GoodData API client.
```
Comment (liry) [inline README.md:18]: plz describe also gooddata-java-parent

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: README.md (hunk 1 above)
Comment (peter-plochan) [reply README.md:19]: added

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by liry** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-java-merge-compliance-check/179/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-java-fossa-scanning-zuul-el7/19/ : FAILURE in 1m 45s (non-voting)

**Pull request merged**